processor/metricstransformprocessor/Makefile

```diff
@@ -0,0 +1 @@
+include ../../Makefile.Common
```

Comment:
It doesn't really matter, but its cleaner (and can prevent issues with some very old tools) to consistently leave a new line at the end of every file